stdlib/public/Concurrency/Error.cpp

```diff
@@ -0,0 +1,19 @@
+//===--- Error.cpp - Error handling support code --------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2017 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+#include "Error.h"
+
+// swift::fatalError is not exported from libswiftCore and not shared, so define another
+// internal function instead.
+SWIFT_NORETURN void swift::swift_Concurrency_fatalError(uint32_t flags, const char *format, ...) {
+  abort();
```

Comment:
Huh. I see that this was just `abort()` before, but it's unfortunate that we didn't even try to report the error message in a usable way.

Reply:
Should we copy the implementation of swift::fatalError in swiftCore?
https://github.com/apple/swift/blob/43c5d8954b2c9bc5a47d690d9d1a68881c540062/stdlib/public/runtime/Errors.cpp#L363-L405